models/infrastructure_edge.py

```diff
@@ -1757,22 +1833,126 @@ async def generate_continents_countries(client: InfrahubClient, log: logging.Log
     log.info("Created continents and countries")
 
 
-async def prepare_accounts(client: InfrahubClient, log: logging.Logger, branch: str, batch: InfrahubBatch) -> None:
-    groups = await client.filters(branch=branch, kind=CoreAccountGroup, name__value="Super Administrators")
-    store.set(key=groups[0].name, node=groups[0])
+async def prepare_permissions(client: InfrahubClient, log: logging.Logger, branch: str, batch: InfrahubBatch) -> None:
+    for p in GLOBAL_PERMISSIONS:
+        try:
+            obj = await client.get(
+                branch=branch, kind="CoreGlobalPermission", hfid=[p.action, str(p.decision)], raise_when_missing=True
+            )
+        except NodeNotFoundError:
+            obj = await client.create(branch=branch, kind="CoreGlobalPermission", data=p.model_dump())
+            batch.add(task=obj.save, node=obj)
+        store.set(key=p.action, node=obj)
+
+    for name, p in OBJECT_PERMISSIONS.items():
+        try:
+            obj = await client.get(
+                branch=branch, kind="CoreObjectPermission", hfid=[p.namespace, p.name, p.action, str(p.decision)]
+            )
+        except NodeNotFoundError:
+            obj = await client.create(branch=branch, kind="CoreObjectPermission", data=p.model_dump())
+            batch.add(task=obj.save, node=obj)
+        store.set(key=name, node=obj)
```

Comment:
We could use upsert mutations here to make it a bit easier to read when creating or ensuring that these exist. Not a blocker for now but perhaps an improvement for the next iteration. Do we need to create the global permissions as those should be created automatically?